content/resources/use-cases/gdm/index.md

```diff
@@ -0,0 +1,154 @@
+relationship was seen, whereby as the predicted ecological distance between
+sites increased (predictors), so too did observed compositional dissimilarity
+(Figure 2). Of the environmental predictors, the sum of I-spline coefficients
+was highest for the predictor value tree cover, indicating the greatest effect
+on compositional dissimilarity. It was seen by the gradient of the slope that
+changes between 0 to 30% tree cover resulted in the greatest changes in
+predicted dissimilarity between study site pairs (Figure 2). However, a
+permutation assessment showed that the model was not statistically significant
+(p = 0.370). The intercept of the model was 0.206, which represents the expected
+dissimilarity between sites when the environmental predictors are equal.
+
+{{% figure src="./GDM_fig2.png" caption="Figure 2: Main output plots of the GDM. A: observed compositional dissimilarity as a function of predicted ecological distance, where each point represents a site pair, and the line represents the GDM-predicted dissimilarity. B: Observed compositional dissimilarity as a function of predicted compositional dissimilarity, where the line represents a line of equality. C -- F: Plotted GDM spline functions for each environmental predictor variable." width="100%"%}}
+
+## Discussion
+
+These results successfully demonstrate a simple workflow in which acoustic
+datasets can be leveraged to provide insight into patterns of biodiversity
+change relative to environmental predictors. In this case, the original sampling
+design could have limited the model's ability to predict changes in composition
+throughout the landscape relative to the environmental predictors. For example,
+all sensors were located inside a patch of vegetation at least 150 m or greater
+in diameter. The total number of sample sites was also small, and sensors were
+positioned across a relatively small spatial. Nevertheless, this is a great
+example workflow of how annotated acoustic data can be used to perform more
+robust modelling methods, such as GDM.
+
+In this example, study site vegetation survey data was available at the location
+of each recorder. However, it is also possible to use spatially complete gridded
+environmental data with the GDM package, such as remotely sensed data in raster
+format. This allows projection of the model across the spatial grid, to map
+predicted patterns of dissimilarity.
+
+With the growing body of acoustic data, coupled with remotely sensed data or
+ground-survey metadata, it will be possible to examine patterns of biodiversity
+variation across large spatial and temporal scales. Identifying environmental
+factors that are important drivers for species occurrences can help to guide
+land management, landscape rehabilitation and inform restoration guidelines. It
+is also possible to make predictions about compositional turnover related to
+changes in environmental conditions, such as land cover, or temperature (climate
+change).
```

Comment:
and an acknowledgements section. Link to ecocommons, mention other resources used again, grants etc...